sklearn/linear_model/tests/test_ridge.py

```diff
@@ -692,6 +692,36 @@ def test_sparse_design_with_sample_weights():
                                       decimal=6)
 
 
+def test_ridgecv_alphas():
+    # Test that no error is raised when fitting RidgeCV
+    # with integer alpha values and also
+    # check that an error is raised for negative alphas
+
+    X = np.array([[-1.0, -1.0], [-1.0, 0], [-.8, -1.0],
+                  [1.0, 1.0], [1.0, 0.0]])
+    y = [1, 1, 1, -1, -1]
+
+    # Integers
+    alphas = (1, 10, 100)
+    ridge = RidgeCV(alphas)
```

Comment:
put directly a list when instantiating: `RidgeCV(alphas=[1, 10, 100])`.
You could also make sure that a numpy array with integer is also converted. In this case use a parametrized test

```python
@pytest.mark.parametrize(
     "alphas",
    [(np.array([1, 10, 100])),
     ([1, 10, 100])])
def test_ridge_cv_alphas(alphas):
    X = ...
    y = ...
    ridge = RidgeCV(alphas)
    ridge.fit(X, y)
```